frontend/tests/e2e/tests/groupmember/AddDependentTest.js

```diff
@@ -0,0 +1,159 @@
+import AddDependentPage from '../../pages/groupmember/AddDependentPage'
+import AlertPage from '../../pages/AlertPage'
+import { SITE_UNDER_TEST } from '../../configuration'
+import { regularAccUser } from '../../roles/roles'
+
+const relationshipOption = AddDependentPage.relationshipSelect.find('option')
+
+const ERROR_MESSAGE = 'Please correct errors before submitting'
+const GROUP_NUMBER_REQUIRED_MESSAGE = 'Group Number is required'
+const INVALID_GROUP_NUMBER_ERROR_MESSAGE = 'Group Number is invalid'
+const GROUP_MEMBER_PHN_REQUIRED_MESSAGE = "Group Member's PHN is required"
+const INVALID_GROUP_MEMBER_PHN_ERROR_MESSAGE = 'PHN format is invalid'
+const DEPENDENT_PHN_REQUIRED_MESSAGE = "Dependent's PHN is required"
+const INVALID_DEPENDENT_PHN_ERROR_MESSAGE = 'PHN format is invalid'
+const RELATIONSHIP_REQUIRED_MESSAGE = 'Relationship is required'
+const IS_STUDENT_REQUIRED_MESSAGE = 'Is this Dependent attending a Canadian Educational Institution? is required'
+const STUDENT_END_DATE_REQUIRED_MESSAGE = 'The value is required'
+const RAPID_RESPONSE = 'RPBS9145 PHN NOT FOUND'
+
+const PAGE_TO_TEST = SITE_UNDER_TEST + '/groupmember/AddDependent'
+
+fixture(`AddDependent Page`).disablePageCaching`Test AddDependent`
+  .beforeEach(async (t) => {
+    await t.useRole(regularAccUser)
+  })
+  .page(PAGE_TO_TEST)
+
+test('Check required fields validation', async (t) => {
+  await t
+    // Given required fields aren't filled out (Group Number, Coverage Effective Date, Group Member's PHN, Dependent's PHN, Relationship, Is Student)
+    // When I click the submit button
+    .click(AddDependentPage.submitButton)
+    // I expect an error message stating the page had errors and individual error messages for each required field
+    .expect(AlertPage.alertBannerText.textContent)
+    .contains(ERROR_MESSAGE)
```

Comment:
I know it's a Prettier thing but I find it much easier to read when the assertion is all on one line.

Reply:
Yeah, it probably is. We can look at turning off Prettier for certain files with .prettierignore if needed.